src/blocks/blocks/lottie/block.json

```diff
@@ -35,7 +35,17 @@
 			"default": false
 		},
 		"width": {
-			"type": "number"
+			"type": [ "string" , "number" ]
```

Comment:
@arinaturcu Didn't know you could do that. Does this work? Thanks, learned something new.

Reply:
Yes, it seems to work. I was a bit surprised too :smile:

Reply:
:0, I can not believe we can use union types.